lambda/js/src/main/scala/feral/lambda/IOLambdaPlatform.scala

```diff
@@ -26,11 +26,18 @@ import scala.scalajs.js.|
 private[lambda] trait IOLambdaPlatform[Event, Result] {
   this: IOLambda[Event, Result] =>
 
-  // @JSExportTopLevel("handler") // TODO
-  final def handler(event: js.Any, context: facade.Context): js.Promise[js.Any | Unit] =
-    (for {
-      lambda <- setupMemo
-      event <- IO.fromEither(decodeJs[Event](event))
-      result <- lambda(event, Context.fromJS(context))
-    } yield result.map(_.asJsAny).orUndefined).unsafeToPromise()(runtime)
+  def main(args: Array[String]): Unit = {
+    val handlerName = getClass.getSimpleName.init
+    js.Dynamic.global.exports.updateDynamic(handlerName)(handlerFn)
```

Comment:
This would be roughly equivalent to `@JSExportTopLevel(handlerName)` annotation on `handlerFn`.